src/app/helpers/page/pageInfo.ts

```diff
@@ -29,7 +30,7 @@ export interface IPageInfo extends Data {
  * Page info class
  */
 export class PageInfo implements IPageInfo {
-  public pageRoute: MenuRoute & { pageComponent?: Type<PageComponent> };
+  public menuRoute: MenuRoute;
```

Comment:
I'm going to suggest this name change is not good.

The point of this field is to store the route for the page. `pageRoute` was descriptive of that function.

I'm still skeptical we even need the whole `MenuRoute` type here. Where else do we depend on the information in `pageInfo.menuRoute` except for getting the route in `get route()` below?

Reply:
The `SecondaryMenuComponent` makes use of the `MenuRoute` stored in the pageInfo. This is required because it needs to know what the current routes `MenuRoute` is, and all parent `MenuRoute`s, so that it can add them to the menu. Here is the relevant code: https://github.com/QutEcoacoustics/workbench-client/blob/master/src/app/components/shared/secondary-menu/secondary-menu.component.ts#L54-L79. `ActionMenuComponent` doesn't need this because we provide the list of menu items in the `menu` value.

Reply:
Ok; it still seems weird but I won't block on that. Maybe in the future we should consider how we've coupled the menu and route. Perhaps this is best where `route` and `menu` are stored as separate fields, even if the contain overlapping information.

I think the reasoning for naming holds though. Agree?

Reply:
I don't mine changing to pageRoute